.github/workflows/callable-flex-update.yml

```diff
@@ -47,5 +47,8 @@ jobs:
                     git rm -q *.json
                     mv .github/flex-endpoint/*.json .
                     git add *.json
+                    # merge in new archived files
+                    for file in ../flex_endpoint/archived/*/*.json; do mv "$file" "archived/$(basename $(dirname "$file"))/$(basename "$file")"; done
```

Comment:
where does `flex_endpoint` come from?
shouldn't this be `mkdir -p archived; mv .github/flex-endpoint/archived/*.json archived/` instead?

Reply:
Ha! Yes, sorry - you're right - this was a local testing path.

So I've added the `mkdir -p archived` (that was missing)Ha! Yes, sorry - you're right - this was a local testing path.

I've added the `mkdir -p archived` (that was missing). I've also fixed the path.

However, I don't think the `mv` you suggested works because the `.json` files are in subdirectories - e.g. they are `archived/symfony.twig-bundle.3.3/e13a79427495e670ef7a19f0e25fd13fbdd909d4.json`. It IS very possible that my mv statement can be simplified, let me know if you have an idea :)